### PR TITLE
fix: adjust bindings for node info

### LINKED ISSUE #3176: [Task]: Re-align types and logic around node info
### Task description

We need to start using the updated node info type.

### Requirements

- [x] Change the API wrapper function to use new bindings
- [x] Change function signatures as needed
- [x] Use the updated node info type

### Acceptance criteria

Meets the above requirements.

### Creation checklist

- [X] I have assigned this task to the correct people
- [X] I have added the most appropriate labels
- [X] I have linked the correct milestone and/or project

### PR DESCRIPTION
**Pull request by maxwellmattryan**
## Summary
Previously, we were using the older node info type.

### Changelog
```
- Changed the API wrapper function to use new bindings
- Changed function signatures as needed
- Updated node info type
```

## Relevant Issues
Closed #3176 

## Type of Change
- [ ] __Breaking__ - any change that would cause existing functionality to not work as expected
- [ ] __Chore__ - refactoring, build scripts or anything else that isn't user-facing
- [ ] __Docs__ - changes to the documentation
- [x] __Fix__ - a change which fixes an issue
- [ ] __New__ - a change which implements a new feature
- [x] __Update__ - a change which updates existing functionality

### Platforms
- __Desktop__
	- [ ] MacOS
	- [x] Linux
	- [ ] Windows
- __Mobile__
	- [ ] iOS
	- [ ] Android

## Checklist
- [x] I have followed the contribution guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added or modified tests that prove my changes work as intended
- [x] I have verified that new and existing unit tests pass locally with my changes
- [x] I have verified that my latest changes pass CI workflows for testing and linting
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: packages/shared/lib/wallet.ts [hunk 1]
```diff
@@ -480,24 +480,13 @@ export async function asyncSyncAccountOffline(account: WalletAccount): Promise<v
     })
 }
 
-export const asyncGetNodeInfo = (accountId: string, url?: string, auth?: IAuth): Promise<INodeInfo> => {
+export const asyncGetNodeInfo = (url?: string, auth?: IAuth): Promise<IStardustNodeInfo> => {
```
Comment (nicole-obrien) [inline packages/shared/lib/wallet.ts:483]: This should be type `INodeInfoResponse` from `node-info-response.interface.ts` in `@core/network`

File: packages/shared/lib/wallet.ts [hunk 2]
```diff
@@ -480,24 +480,13 @@ export async function asyncSyncAccountOffline(account: WalletAccount): Promise<v
...
-            onSuccess(response) {
-                resolve(response.payload)
-            },
-            onError(err) {
-                reject(err)
-            },
-        })
-    })
+    const manager = get(profileManager)
+    return manager.getNodeInfo(url, auth)
```
Comment (nicole-obrien) [inline packages/shared/lib/wallet.ts:489]: async function but we are not awaiting on this?

File: packages/shared/lib/wallet.ts [hunk 3]
```diff
@@ -480,24 +480,13 @@ export async function asyncSyncAccountOffline(account: WalletAccount): Promise<v
     })
 }
 
-export const asyncGetNodeInfo = (accountId: string, url?: string, auth?: IAuth): Promise<INodeInfo> => {
+export const asyncGetNodeInfo = (url?: string, auth?: IAuth): Promise<IStardustNodeInfo> => {
     if (!url || (!url && !auth)) {
```
Comment (nicole-obrien) [inline packages/shared/lib/wallet.ts:484]: This method takes the url and auth as optional so why do we have this guard? If these are passed in then we will just get the current nodes info.

**Review by nicole-obrien — CHANGES_REQUESTED** [review]

File: packages/shared/lib/wallet.ts (hunk 2 above)
Comment (maxwellmattryan) [reply packages/shared/lib/wallet.ts:489]: It's not a in a try / catch block so doesn't need the `await` keyword.

**Review by nicole-obrien — APPROVED** [review]

**Pull request merged**